### PR TITLE
Edit external node guide

### PR DESCRIPTION
**Pull request by siljapetasch**
Edited our external node guide. Also added a short section about how to monitoring Sync Progress.

### CODE REVIEW

File: pages/guides/run-a-node/op-stack-node.mdx [hunk 1]
```diff
@@ -1,29 +1,60 @@
...
 
-and `OP_NODE_P2P_STATIC` via
+Repeat the same for `OP_NODE_P2P_STATIC` with the following API call:
 
 ```txt copy
 https://api.conduit.xyz/public/network/staticPeers/<network-slug>
 ```
 
-For `OP_GETH_SEQUENCER_HTTP`, you can use the public RPC of your network. For production usage, you can get an API Key with [Conduit Nodes](https://conduit.xyz/nodes).
+Consider setting `P_NODE_P2P_SYNC_ONLYREQTOSTATIC` to `true` if network reliability is a concern, especially when dealing with networks that may have unreliable peers. This setting ensures more consistent and stable synchronization by forcing the node to request sync data only from `OP_NODE_P2P_STATIC`.
```
Comment (optix2000) [inline pages/guides/run-a-node/op-stack-node.mdx:46]: Typo in `P_NODE_P2P_SYNC_ONLYREQTOSTATIC` should be `OP_NODE`

File: pages/guides/run-a-node/op-stack-node.mdx [hunk 2]
```diff
@@ -1,29 +1,60 @@
...
+
+#### OP-GETH Sequencer HTTP
```
Comment (optix2000) [inline pages/guides/run-a-node/op-stack-node.mdx:48]: Might be good to elaborate that this is required if you plan on submitting transactions to your node. Right now it sounds ambiguous what it's used for. Also good to point out that you can find said public RPC URL in the dashboard.

File: pages/guides/run-a-node/op-stack-node.mdx [hunk 3]
```diff
@@ -1,29 +1,60 @@
...
-To run a self-hosted node for an OP Stack rollup deployed on Conduit, please follow [Optimism's documentation](https://docs.optimism.io/builders/node-operators/tutorials/node-from-docker).
+To run a self-hosted node for an OP Stack rollup deployed on Conduit, please follow the steps below to configure the necessary settings and gather all required information. Afterwards, you can proceed with [Optimism's documentation](https://docs.optimism.io/builders/node-operators/tutorials/node-from-docker) on how to run a node.
 
-You can fetch `rollup.json` and `genesis.json` from [Browse Rollups](https://app.conduit.xyz/browse-networks), clicking on `Run a node` for the network.
+<Steps>
+### Sign In
+Log in to the [Conduit app](https://app.conduit.xyz).
+ 
+### Access Settings
+Navigate to your deployment dashboard. Click on `Settings`, then select `General` from the sidebar.
```
Comment (optix2000) [inline pages/guides/run-a-node/op-stack-node.mdx:20]: Screenshots would be good on the next version, but this is good for now.

File: pages/guides/run-a-node/op-stack-node.mdx [hunk 4]
```diff
@@ -1,29 +1,60 @@
...
+#### `rollup.json` and `genesis.json`
+
+In your deployment dashboard, navigate to `Overview`. Click the `Run a Node` button on the right to get `rollup.json` and `genesis.json` files.
+
+#### P2P Configuration
+
+To obtain the values for `OP_NODE_P2P_BOOTNODES`, replace `<network-slug>` in the following API call:
 
 ```txt copy
 https://api.conduit.xyz/public/network/bootnodes/<network-slug>
```
Comment (optix2000) [inline pages/guides/run-a-node/op-stack-node.mdx:37]: At some point we should move this to be in the dashboard, but this is fine for now.

Comment (siljapetasch) [reply pages/guides/run-a-node/op-stack-node.mdx:37]: raised this with jeff and bertie in slack, maybe we can already put it in the backlog with low prio

File: pages/guides/run-a-node/op-stack-node.mdx (hunk 1 above)
Comment (siljapetasch) [reply pages/guides/run-a-node/op-stack-node.mdx:46]: corrected

File: pages/guides/run-a-node/op-stack-node.mdx (hunk 2 above)
Comment (siljapetasch) [reply pages/guides/run-a-node/op-stack-node.mdx:48]: adjusted

**Review by optix2000 — APPROVED** [review]
lgtm

**Pull request merged**